### ISSUE TITLE
Add a zoom  feature to the waterfall plot script

### ISSUE THREAD

**Issue by hughbg** [hughbg/leda_analysis_2016#4]
So we can look close at specific frequency ranges or time intervals in the waterfall plots. Also look at TV channel signals.

**Comment by hughbg** [comment]
It's possible to zoom into a plot created by matplotlib and displayed on the screen, which applies to all the plots generated by the Python scripts. However the zoom does not increase the resolution of what you are looking at, it just zooms the existing image.

Low priority.

**Comment by jaycedowell** [comment]
9742aef does something like this.  Is that what you were envisioning?

**Comment by hughbg** [comment]
Yes it will improve the waterfall plots as a way of quick assessment. The plot_ds9.py script was created for looking at data at high resolution, because we can use the functionality of DS9 if we can generate a FITS file of the waterfall plot. This only allows viewing of 1 antenna at a time though.
